### PR TITLE
Add route to list available projects and workflows

### LINKED ISSUE #12: Is it possible to get a list of available projects or workflows
I know that I can get these from provenence, but it would be helpful if i could query the API in some way to get a list of what is available

### PR DESCRIPTION
**Pull request by callunity**
Fixes #12 .

### CODE REVIEW

File: test/fileqcTest.js [hunk 1]
```diff
@@ -202,11 +206,46 @@ describe('FileQcController', () => {
...
 
+describe('available constants', () => {
+  beforeEach(async () => {
+    await cmd.run(
+      'sqlite3 ' +
+        process.env.SQLITE_LOCATION +
+        '/fpr.db < ' +
+        test_fpr_creation
+    );
+    await cmd.run(
```
Comment (apmasell) [inline test/fileqcTest.js]: Why is this duplicated?

Comment (callunity) [reply test/fileqcTest.js]: It isn't quite duplicated -- the first one is the FPR creation file, the second one is the migration file.

Comment (apmasell) [reply test/fileqcTest.js]: Oh. I see now. Sorry.

Comment (callunity) [reply test/fileqcTest.js]: No worries - the formatting made that less obvious!

**Pull request merged**